axon-framework/messaging-concepts/message-intercepting.md

```diff
@@ -52,6 +52,8 @@ Axon Framework has support for JSR 303 Bean Validation based validation. This al
 
 The `BeanValidationInterceptor` also implements `MessageHandlerInterceptor`, allowing you to configure it as a handler interceptor as well.
 
+> It is important to note that this version of the `BeanValidationInterceptor` is only usable with `javax`. If you are moving to a `jakarta` approach of bean validation, you should use the `JakartaBeanValidationInterceptor`. 
```

Comment:
A short title for this notice would be nice. :-)

Reply:
hahaha... I knew you were going to ask for it... I gave it some thought yesterday and couldn't come up with a good one. Let's see if this one is good.

Reply:
Looks good to me!